notebook/static/notebook/js/codecell.js

```diff
@@ -313,7 +313,7 @@ define([
             stop_on_error = true;
         }
 
-        this.output_area.clear_output(false, true);
+        this.clear_output(false, true);
```

Comment:
So I think this diff should make it into the notebook via the PR or https://github.com/jupyter/notebook/pull/2411